docs/source/en/package_reference/file_download.md

```diff
@@ -34,5 +34,5 @@ The methods displayed above are designed to work with a caching system that prev
 re-downloading files. The caching system was updated in v0.8.0 to become the central
 cache-system shared across libraries that depend on the Hub.
 
-Read the [cache-system guide](../manage-cache) for a detailed presentation of caching at
+Read the [cache-system guide](../how-to-cache) for a detailed presentation of caching at
```

Comment:
```suggestion
Read the [cache-system guide](../guides/manage-cache) for a detailed presentation of caching at
```

Even though we have a redirect `how-to-cache` => `guides/manage-cache` (for legacy links) it's best to use the new one to avoid a redirect for the user.

Reply:
Thank you!